assets/ui.css

```diff
@@ -1,3 +1,14 @@
+.close-button{
+  font-weight: bold;
+  float: left;
+  cursor: pointer;
+  margin-left: 0.5%;
```

Comment:
Maybe a fixed value, as there is no need to change the value based on window size.